### PR TITLE
Remove Swarm

### PR DESCRIPTION
**Pull request by NicolasT**
Some users tried to deploy Zenko on Docker Swarm recently using the provided manifests, which failed. It appears the Swarm deployment system hasn't been updated for some time, and all focus has been on Kubernetes deployment.

Let's not confuse users and remove all Swarm-related code and documentation.

Closes: #1128

### CODE REVIEW

**Review by rahulreddy — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -28,10 +28,7 @@ If you'd like to contribute, please review the
 ## Overview
 
 This repository includes installation resources to deploy the full Zenko
-stack over different orchestration systems.
-
-Currently we have [Kubernetes](https://kubernetes.io/) and
-[Docker Swarm](https://docs.docker.com/engine/swarm/).
+stack on the [Kubernetes](https://kubernetes.io/)  orchestration system.
```
Comment (miniscruff) [inline README.md]:
```suggestion
stack on the [Kubernetes](https://kubernetes.io/) orchestration system.
```
There is a double space if you care to delete it, it doesn't seem to affect the rendered output though.

Comment (NicolasT) [reply README.md]: Done, though if CI doesn't catch it I should've ignored it :wink:

**Comment by NicolasT** [comment]
@bert-e reset

File: README.md (hunk 1 above)
Comment (miniscruff) [reply README.md]: I don't think I have ever seen any linting catch double spaces, it is just something I notice in reviews.

**Review by miniscruff — APPROVED** [review]

**Review by tcarmet — APPROVED** [review]

**Comment by NicolasT** [comment]
Looks like CI for 1.2 is broken:
```
Adding stable repo with URL: https://kubernetes-charts.storage.googleapis.com 
Error: error initializing: Looks like "https://kubernetes-charts.storage.googleapis.com" is not a valid chart repository or cannot be reached: Failed to fetch https://kubernetes-charts.storage.googleapis.com/index.yaml : 403 Forbidden
```
Indeed, this repo is no longer available, and one should use `helm init --stable-repo-url https://charts.helm.sh/stable ...` instead. However, very much outside the scope of this PR. I'd go as far to suggest a test bypass given #1152 did succeed.

**Pull request merged**